assets/js/shortcuts.js

```diff
@@ -98,15 +98,24 @@ function setupShortcuts(shortcutDepth = 2) {
 
   shortcutDepth += 1; // to account for the page title
 
-  // Build a class selector for each header type, and concatenate with commas
+  /*
+   * We have build selectors to find headers in different contexts.
+   * i. direct children of .content-container (main structure)
+   * i. headers within tabpanels (for tabbed content)
```

Comment:
I thought these were the ones we explicitly avoided before? We don't want to link to the content of tabpanels!